README.md

```diff
@@ -1,53 +1,56 @@
-# Award weighted criteria extension for EU data
-
-> OCDS extension to describe criteria as expressed in eForms (BG-707) and compatible with TED
-
-This extension aims primarily at supporting criteria data formatted according to eForms data structure, but it also aims at enabling the OCDS publication of TED v2.0.9 data.
-
-## Guidance for TED mapping
-
-If only `AC_QUALITY` is set,  set `tender.awardCriteria` to "qualityOnly".
-
-If only `AC_COST` is set, set `tender.awardCriteria` to "costOnly".
-
-If only `AC_PRICE` is set, set `tender.awardCriteria` to "priceOnly".
-
-If a mix of `AC_QUALITY`, `AC_COST` and `AC_PRICE` is set, set `tender.awardCriteria` to "ratedCriteria".
-
-Add an `AwardWeightedCriteria` object to the `tender.awardWeightedCriteriaPerLot` array.
-
-Set the `AwardWeightedCriteria` object as follow:
-
-**AC_QUALITY**
-
-Add an `AwardWeightedCriterion` object to the `awardWeightedCriteria` array.
-
-Map `AC_CRITERION` to `criterionName`.
-
-Map `AC_WEIGHTING` to `criterionDescription`.
-
-Set `criterionType` to "quality".
-
-**AC_COST**
-
-Add an `AwardWeightedCriterion` object to the `awardWeightedCriteria` array.
-
-Map `AC_CRITERION` to `criterionName`.
-
-Map `AC_WEIGHTING` to `criterionDescription`.
-
-Set `criterionType` to "cost".
-
-**AC_PRICE**
-
-Add an `AwardWeightedCriterion` object to the `awardWeightedCriteria` array.
-
-Map `AC_CRITERION` to `criterionName`.
-
-Map `AC_WEIGHTING` to `criterionDescription`.
-
-Set `criterionType` to "price".
-
-**AC_PROCUREMENT_DOC**
-
-Discard.
+# Award criteria breakdown
+
+Adds an award criteria breakdown array to the tender object, to break down award criteria by price, cost and quality.
+
+## Legal context
+
+In the European Union, this extension's fields correspond to [eForms BG-707 (Award Criteria)](https://github.com/eForms/eForms). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
+
+[Directive 2014/24/EU](https://eur-lex.europa.eu/eli/dir/2014/24/oj) [Article 67](https://eur-lex.europa.eu/eli/dir/2014/24/oj#d1e5950-65-1)(5) describes weightings and orders of importance.
+
+## Example
+
+```json
+{
+  "tender": {
+    "awardCriteriaBreakdown": [
+      {
+        "relatedLots": [
+          "1",
+          "2"
+        ],
+        "awardCriteria": [
```

Comment:
Could you add an example with `fixed`? I don't understand how it's used.

Reply:
Yes. Otherwise, the changes look good?

Reply:
I added an example, based on https://www.felp.ac.uk/content/most-economically-advantageous-tender-meat and https://github.com/eprocurementontology/eprocurementontology/issues/157